site/content/docs/5.1/forms/quantity-selector.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: docs
+title: Quantity selector
+description: Use our custom quantity selector in forms for selecting a number.
+group: forms
+toc: true
+---
+
+## Default
+
+Quantity selector is a form element used to select a number.
+
+**Please note that this component is suitable for incremental and decremental display of small numbers, i.e. numbers with 3-4 digits. We do not recommend to use it with larger numbers.**
+
+A default value can be specified with the input `value` attribute.
+
+Value will vary between the values defined for the `min` and `max` attributes (negatives values are allowed). `min` and `max` values are customizables.
```

Comment:
* I think it is rather "negative value" instead of "negatives values"
* "customizables" → "customizable"